tests/bid_tests.rs

```diff
@@ -455,31 +455,31 @@ mod protocol_tests {
         let latest_consensus_round = BlsScalar::from(200u64);
 
         let mut circuit = BlindBidCircuit {
-            bid: Some(bid),
-            score: Some(score),
-            secret_k: Some(secret_k),
-            secret: Some(secret),
-            seed: Some(consensus_round_seed),
-            latest_consensus_round: Some(latest_consensus_round),
-            latest_consensus_step: Some(latest_consensus_step),
-            branch: Some(&branch),
-            size: 0,
-            pi_constructor: None,
+            bid: bid,
+            score: score,
+            secret_k: secret_k,
+            secret: secret,
+            seed: consensus_round_seed,
+            latest_consensus_round: latest_consensus_round,
+            latest_consensus_step: latest_consensus_step,
+            branch: &branch,
+            trim_size: 1 << 15,
+            pi_positions: vec![],
         };
 
-        let (pk, vk, _) = circuit.compile(&pub_params)?;
-        let proof = circuit.gen_proof(&pub_params, &pk, b"NonElegibleBid")?;
+        let (pk, vk) = circuit.compile(&pub_params)?;
+        let proof = circuit.gen_proof(&pub_params, &pk, b"vec![]legibleBid")?;
```

Comment:
This looks like a mishap with 'Replace All'

Reply:
Ty